packs/feats/game-hunter-dedication.json

```diff
@@ -79,7 +79,7 @@
                     }
                 ],
                 "selector": "attack",
-                "text": "When you succeed at a Strike against your hunted prey while it's off-guard, it must attempt a Fortitude save against your class DC. On a failure, the prey's Speeds are each reduced by 10 feet for 1 round; on a critical failure, the duration of this effect is 1 minute. The prey is then temporarily immune to this effect for 10 minutes."
+                "text": "PF2E.SpecificRule.GameHunterDedication.SuccessAttack"
```

Comment:
Consider two Note REs - one for critical success, one for success.

Reply:
Notes are the same for success and critical, because it's attack

After succeed attack - target roll saving throw and based on result need to set effect with 1 round/minute